### DISCUSSION TITLE
How can I get badge?

### DISCUSSION THREAD

**Discussion by kairos-hk** [community/community#27990] [Other Feature Feedback, Questions, & Ideas]
How can I get Pull Shark badge?

**Comment by radar-blip** [comment]
https://github.com/Schweinepriester/github-profile-achievements#achievements

**Comment by jfullstackdev** [comment]
from my own badge it says

![image](https://user-images.githubusercontent.com/47092464/182288666-d7e489fd-4e26-43a9-9750-3b6d4a40c28c.png)

**Reply by kairos-hk** [reply]
Thank you

**Comment by DSDmark** [comment]
To get Achieve that badge you need to do at least 2 pull requests merged in your repo

> You're able to get the Pull Shark Achievement by opening pull requests that eventually are merged with a repo.
